src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,269 @@
+import * as clipboard from 'clipboard-polyfill/text'
+import _ from 'lodash/fp'
+import { useEffect, useState } from 'react'
+import { div, h, span, table, tbody, td, tr } from 'react-hyperscript-helpers'
+import ReactJson from 'react-json-view'
+import * as breadcrumbs from 'src/components/breadcrumbs'
+import Collapse from 'src/components/Collapse'
+import { Link } from 'src/components/common'
+import { centeredSpinner, icon } from 'src/components/icons'
+import {
+  collapseCromwellExecutionStatus,
+  failedIcon,
+  makeSection,
+  makeStatusLine,
+  runningIcon,
+  statusIcon, submittedIcon,
+  successIcon, unknownIcon
+} from 'src/components/job-common'
+import UriViewer from 'src/components/UriViewer'
+import WDLViewer from 'src/components/WDLViewer'
+import { Ajax } from 'src/libs/ajax'
+import { bucketBrowserUrl } from 'src/libs/auth'
+import { getConfig } from 'src/libs/config'
+import { withErrorReporting } from 'src/libs/error'
+import * as Nav from 'src/libs/nav'
+import * as Style from 'src/libs/style'
+import * as Utils from 'src/libs/utils'
+import { wrapWorkspace } from 'src/pages/workspaces/workspace/WorkspaceContainer'
+
+
+const styles = {
+  sectionTableLabel: { paddingRight: '0.6rem', fontWeight: 600 }
+}
+
+const groupCallStatuses = callsObject => {
+  const statusCounts = {}
+  for (const callname in callsObject) {
+    for (const attempt in callsObject[callname]) {
+      const executionStatus = collapseCromwellExecutionStatus(callsObject[callname][attempt].executionStatus)
+      statusCounts[executionStatus] = (statusCounts[executionStatus] || 0) + 1
+    }
+  }
+  return statusCounts
+}
```

Comment:
This should probably be rewritten using some combination of `_.reduce` and `_.map`. I'd be happy to talk through it with you, @panentheos can probably give you a more reasonable idea of how to start faster than I can, tho.

Reply:
I think this could be:
```suggestion
const groupCallStatuses = _.flow(
  _.values,
  _.flattenDepth(1),
  _.countBy(a => collapseCromwellExecutionStatus(a.executionStatus))
)
```